walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -121,6 +123,51 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void moreTests1() {
```

Comment:
The title of the test should be more clear. Usually we use a title like this "<the function I test>_<the special condition of testing>_<the expected result>". So this test can be renamed to 

public void findMeetingQuery_shouldReturnValidStartEndTime() {
}

My suggestion might not be the best naming for the test. But we should have more clear test names.

Reply:
I don't think these tests have anything especially unique I can identify easily in the function name. Instead I've kept the names and added a comment above each one to describe them in more detail.